### PR TITLE
IPC Surgery Fix

### PR DESCRIPTION
**Pull request by Aurorablade**
fixes https://github.com/ParadiseSS13/Paradise/issues/7974

:cl: Fethas
bugfix: Dethaws some IPC surgery issues.
/:cl:

### CODE REVIEW

File: code/modules/surgery/other.dm [hunk 1]
```diff
@@ -245,8 +245,10 @@
 	possible_locs = list("head", "chest", "groin")
 
 /datum/surgery/remove_thrall/synth
+	name = "Debug Shadow Tumor"
```
Comment (tigercat2000) [inline code/modules/surgery/other.dm]: debug message?

Comment (Aurorablade) [inline code/modules/surgery/other.dm]: no..clever play on a surgery name for IPCs....

Comment (tigercat2000) [inline code/modules/surgery/other.dm]: Kek. I see.

Comment (taukausanake) [inline code/modules/surgery/other.dm]: Doesn't debugging something mean it's kind of supposed to be there? I mean I can debug Paradise code (barely) but that doesn't mean it's getting a full removal like the tumor is going to. Unless I'm missing something wouldn't "deletion" be a better choice?

Comment (Crazylemon64) [inline code/modules/surgery/other.dm]: I am assuming it's "Debug" as in "remove abberant behavior from the posibrain"

**Pull request merged**